python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -89,7 +89,9 @@ async def setup_calsys(self, calib_type: str) -> None:
         The dome and telescope can be moving into place concurrently.
 
         """
-        raise NotImplementedError()
+        await self.warmup_lightsource(calib_type)
```

Comment:
Again, if this method is just calling another method from the class, one of them should be removed.

Reply:
Done. I thought I'd add another in case we wanted to move the telescope and dome as part of this method.